plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -413,4 +432,8 @@ protected String getColumnComment(String tableName, String columnName)
     protected abstract void dropTableFromMetastore(String tableName);
 
     protected abstract String getMetadataLocation(String tableName);
+
+    protected abstract String schemaPath();
+
+    protected abstract boolean locationExists(String location);
```

Comment:
#16032 added another smoke test to Iceberg connector. Could you rebase and override these methods in `TestIcebergAbfsConnectorSmokeTest`? Sorry for bothering you.

Reply:
No issue :) I have overridden methods in `TestIcebergAbfsConnectorSmokeTest`.